src/watchmaker/workers/salt.py

```diff
@@ -661,6 +667,49 @@ def _set_grain(self, grain, value):
         self.log.info('Setting grain `%s` ...', grain)
         super(SaltLinux, self)._set_grain(grain, value)
 
+    def _install_pip_modules(self):
+        if self.pip_install:
+            python = self.run_salt(['grains.get', 'pythonexecutable'])
+            py_exec = python['stdout'].decode().split('\n')[1].strip()
+            self.log.info('Salt Python interpreter: `%s`', py_exec)
+
+            try:
+                ensure = self.call_process([py_exec, '-m', 'ensurepip'])
+                self.log.info('Ensure pip: %s', ensure['stdout'])
+                ver = self.call_process([py_exec, '-m', 'pip', '--version'])
+                self.log.info('Pip version: %s', ver['stdout'])
+            except WatchmakerException:
+                self.log.info('Pip not installed for Salt interpreter!')
+                self.log.info('Attempting pip install...')
+                get_pip = [
+                    'curl',
+                    'https://bootstrap.pypa.io/get-pip.py',
```

Comment:
This won't work on other networks. It will need to be a new config value that can _default_ to this (or for better compatibility, to the version that works with py26).

Reply:
Using vendored file